inbm/dispatcher-agent/dispatcher/aota/application_command.py

```diff
@@ -135,27 +135,15 @@ def update(self) -> None:
             # Move driver to CentOS filesystem
             shutil.move(driver_path, driver_centos_path)
 
-            # Remove old driver
-            old_driver_name = self.identify_package(driver_path.split('/')[-1])
-            if not old_driver_name:
-                raise AotaError(
-                    f'AOTA Command Failed: Unsupported driver {driver_path.split("/")[-1]}')
-            uninstall_driver_cmd = DOCKER_CHROOT_PREFIX + \
-                f'/usr/bin/rpm -e --nodeps {old_driver_name}'
-            out, err, code = PseudoShellRunner().run(uninstall_driver_cmd)
-            logger.debug(out)
-            # If old packages wasn't install on system, it will return error too.
-            if code != 0 and "is not installed" not in str(err):
-                raise AotaError(err)
-
             chroot_driver_path = driver_centos_path.replace("/host", "")
-            install_driver_cmd = DOCKER_CHROOT_PREFIX + f'/usr/bin/rpm -ivh {chroot_driver_path}'
+            install_driver_cmd = CHROOT_PREFIX + \
```

Comment:
This needs to be DOCKER_CHROOT_PREFIX in case the RPM wants to restart services.

Reply:
Hi @gblewis1 , the reason I changed back to CHROOT_PREFIX is the error below observed during the passing scenario. Inside inb container, no /usr/bin/docker has been found. 

![image](https://user-images.githubusercontent.com/58315865/156076702-ac8081da-6d33-439e-9d80-974547ea21a4.png)

Reply:
@yengliong93 Docker binary needs to be installed inside the container. We have that in our existing Dockerfile, but basically you want to install the 'docker.io' package and you want to run 'systemctl disable docker' to disable the service.

Reply:
I see. Let me update the Dockerfile to install docker.io and disable docker.

Reply:
@gblewis1  Using DOCKER_CHROOT_PREFIX, it failed to execute the rpm command. With CHROOT_PREFIX, the rpm command pass. Can we use the CHROOT_PREFIX first in CentOS driver installation and later debug the issue? 

![image](https://user-images.githubusercontent.com/58315865/156096798-f4dbbe94-5dc4-4769-bd32-0bb95e1f3390.png)